src/openassetio-python/package/openassetio/pluginSystem/PythonPluginSystem.py

```diff
@@ -117,9 +117,9 @@ def scan_entry_points(self, entryPointName):
         there was a problem loading importlib_metadata.
         """
 
-        # We opt to use the backport implementation of modern importlib to avoid
-        # needing to support 3+ code paths to cover Python 3.7 to 3.10. It is
-        # made available for 3.7 onwards (for now, at least).
+        # We opt to use the backport implementation of modern importlib
+        # since the API changes from 3.9 to 3.10. It is made available
+        # for 3.7 onwards (for now, at least).
```

Comment:
Although I think the 3.7 reference is to do with the importlib backport, is it worth removing the 3.7 just to purge the sourcetree of any reference? As knowing it's available from 3.7 doesn't aid anything here IMO.

Reply:
Yeah good point

Reply:
Removed in 75abe614